main.cpp

```diff
@@ -8,12 +8,15 @@
 int main(int argc, char *argv[])
 {
     char optc;
+    bool invert_steering = false;
+    bool invert_throttle = false;
+    bool invert_axis = false;
 
     rf_global_args.FREQ = DEFAULT_FREQ;
     rf_global_args.SAMPLE_RATE = DEFAULT_SAMPLE_RATE;
     rf_global_args.SYMBOL_RATE = DEFAULT_SYMBOL_RATE;
 
-    while ((optc = getopt(argc, argv, "f:s:b:")) > 0) {
+    while ((optc = getopt(argc, argv, "f:s:b:STA")) > 0) {
```

Comment:
The rc car I was testing on has the steering backwards (so left in your program turned the car right and right turned it left).  I figured that since the first device I tried had a problem, it wasn't too much of a leap to assume there are other improperly designed devices out there.